scijava/scijava-ops-engine/src/test/java/org/scijava/ops/engine/impl/ProvenanceTest.java

```diff
@@ -358,6 +359,35 @@ public void testAdaptedOpWithDependencies() {
 	 */
 	@Test
 	public void testSimplificationRecovery() {
+		// Get the Op
+		Computers.Arity1<ObjectArray<Number>, ObjectArray<Number>> c = ops //
+				.op("test" + ".provenanceComputer") //
```

Comment:
Why is this string split with the `+`  operator here?

Reply:
It isn't anymore!